src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -0,0 +1,306 @@
+</Callout>
+
+After running the Backend with `cargo run` you should see the following output:
+
+<CodeBlock lang="output">
+
+```s
+2023-07-11T11:54:17.553899587+01:00 [torrust_index_backend::web::api::server][INFO] Starting API server with net config: 0.0.0.0:3001 ...
+2023-07-11T11:54:17.553946897+01:00 [torrust_index_backend::web::api::server][INFO] API server listening on http://0.0.0.0:3001
+```
+
+</CodeBlock>
+
+You should be able to load the API entrypoint on <http://0.0.0.0:3001/>
+
+<Image src="/images/posts/torrust-backend-api-entrypoint-response-screenshot.png" alt="Screenshot of Torrust Backend API entrypoint response" />
+
+For more details about the Torrust Index Backend, check the [Index Backend documentation](https://docs.rs/torrust-index-backend/).
+
+## Set Up the Torrust Frontend
+
+At the time of writing, the Frontend requires:
+
+- Node: `^19.0.0`
+
+The frontend is a [Nuxt](https://nuxt.com/) application.
+
+You can run the Tracker Frontend with the following commands:
+
+<CodeBlock lang="terminal">
+
+```bash
+cd torrust-index-frontend/
+./bin/install.sh
+npm run dev
+```
+
+</CodeBlock>
+
+You should see the following output:
+
+<Image src="/images/posts/
+running-torrust-frontend-in-dev-mode.png" alt="Screenshot of Torrust Index Frontend running from the terminal" />
+
+Go to <http://localhost:3000/torrents> and you should see the torrent list page page:
+
+<Image src="/images/posts/index-screenshot-torrent-list-page.png" alt="Screenshot of torrent list page on the browser" />
+
+For more details about the Torrust Index Frontend, check the Index [Frontend documentation](https://github.com/torrust/torrust-index-frontend).
+
+## Application Setup
+
+There are some public pages like the torrent list or torrent details pages. But there are some pages that require authentication like the admin pages or the pages for uploading or editing torrents.
+
+To access those pages you need to create an account. You can do it from the signup page:
+
+<http://localhost:3000/signup>
+
+<Callout type="info">
+  The first user created will be the <code>admin</code> user.
+</Callout>
+
+## Development tools
+
+We found some tools very useful to develop the Torrust Tracker and Index. We hope you find them useful too.
+
+- DB Browser for SQLite: <https://sqlitebrowser.org/>
+- BitTorrent client: qBittorrent <https://www.qbittorrent.org/>
+- Torrent file editor: imdl <https://github.com/casey/intermodal>
+- DaisyUI component library: <https://daisyui.com/components/>
+
+## Conclusion
+
+As you can see, the Torrust Tracker and Index are very easy to set up. We hope you find this guide useful.You might have problems:
```

Comment:
There is a space missing at the end of the sentence after the full stop of the second sentence.
"....useful. You...."

Reply:
Fixed.